tools/sense_studio/templates/navigation.html

```diff
@@ -0,0 +1,70 @@
+{%  set class_labels = inject_class_labels(project) %}
+<a href="{{ url_for('project_details', project=project) }}" class="item">Project Details</a>
+<div class="ui simple dropdown item">
+    Video Recording
+    <i class="dropdown icon"></i>
+    <div class="menu">
+        <div class="item">
+            <i class="dropdown icon"></i>
+            Train
+            <div class="menu">
+                {% for label in class_labels  %}
+                    <a href="{{ url_for('video_recording_bp.record_video', project=project, split='train',
+                     label=label) }}" class="item">{{label}}</a>
```

Comment:
Weird line break. I don't think we have a character limit on html files yet.
And please add spaces around the variable: `{{  label  }}`